app/styles/newcomponents/ilios-chart-tooltip.scss

```diff
@@ -9,16 +9,18 @@
   margin: 0;
   max-height: 150px;
```

Comment:
We can remove it max-height. The height of the tooltip will scale to fit all the content.